test cases/vala/28 generated ui file/meson.build

```diff
@@ -0,0 +1,34 @@
+project('demo', 'c', 'vala')
+
+gnome = import('gnome')
+
+deps = [
+  dependency('glib-2.0', version : '>=2.50'),
+  dependency('gobject-2.0'),
```

Comment:
I don't think you need to explicitly depend on these. GTK will bring them in for you

Reply:
Meson has special handling for GLib, but dropping gobject is probably fine. Will do.

Reply:
I am not aware of the practice of explicitly depending on `glib-2.0` and `gobject-2.0`. Is there a difference in how meson handles the dependencies?

Reply:
Meson has special handling for GLib (namely automatically passing `--target-glib`). gobject is something vala pretty much requires to be present (on glib profile, anyways), so it makes sense to explicitly depend on it IMO.

Reply:
interesting, that should be added to the templates I guess then.